tools/msconvert/msconvert_macros.xml

```diff
@@ -25,7 +25,9 @@
     #else
       --input=${input}
       #if hasattr($input, 'display_name')
-      --input_name='${input.display_name}'
+      ##--input_name='${input.display_name}'
+      #set basename = $input.display_name
```

Comment:
Blacklisted, just means there is no conda package at the moment and we can not test it with travis. The correct way would be to get this dependency into Conda, or check if there is already one. Thanks @jvolkening!